src/test/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingControllerTest.kt

```diff
@@ -43,8 +42,23 @@ class ArbeidsfordelingControllerTest : OppslagSpringRunnerTest() {
                                                                                                           HttpEntity(null,
                                                                                                                      headers))
 
+        assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+        assertThat(response.body.data).hasSize(1)
+        assertThat(response.body.data!!.first().enhetId).isEqualTo("4820")
+    }
 
+    @Test
+    fun `skal hente geografisk tilknytning og diskresjonskode for ident og kalle arbeidsfordelingklient sin hent enhet - post ident`() {
+        stubFor(post("/graphql")
+                        .willReturn(aResponse()
+                                            .withStatus(200)
+                                            .withHeader("Content-Type", "application/json")
+                                            .withBody(readfile("pdlEnhetResponse.json"))))
 
+        val response: ResponseEntity<Ressurs<List<ArbeidsfordelingClient.Enhet>>> =
+                restTemplate.exchange(localhost(ENHET_URL),
+                                      HttpMethod.POST,
+                                      HttpEntity(PersonIdent(PERSON_IDENT), headers))
```

Comment:
Nå er vel strengt tatt "headers" her potensielt med "Nav-Personident" (kommer an på hvilke tester som kjøres først?) ettersom headers er global for klassen? Burde bruken av headers her være immutable i hver test?

Reply:
Ja, det stemmer nok. Skal se på det!

Reply:
9c62330